src/foam/u2/crunch/CapabilityStore.js

```diff
@@ -189,7 +189,7 @@ foam.CLASS({
     },
     function initE() {
       this.SUPER();
-
+      this.crunchService.sub('updateJunction', this.onChange);
```

Comment:
If you do:
this.onDetach(this..crunchService.sub('updateJunction', this.onChange));
then it will remove the listener automatically when this view is destroyed.